pyproject.toml

```
@@ -21,7 +21,7 @@ select = ["E", "W", "F", "C", "N", "R", "D", "I001"]
 # Ignore E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
 # Ignore W605 as it doesn't recognize raw strings
-ignore = ["E501", "D107", "W605", "RET504"]
+ignore = ["E501", "D107", "W605", "RET504", "C901"]
```

Does toml support "breaking" comments?

```suggestion
ignore = [
  "E501",  # What is this?
  "D107",  # Missing docstring in __init__
  "W605",
  "RET504",
  "C901"
]
```